iocore/eventsystem/I_EThread.h

```diff
@@ -85,6 +85,7 @@ enum ThreadType {
 class EThread : public Thread
 {
 public:
+  static thread_local EThread *this_ethread_ptr;
```

Comment:
Bit of magic - if you do this
```
static inline thread_local EThread *this_ethread_ptr = nullptr;
```
then you don't need to declare it in the ".cc" file.